### PR TITLE
Update script to work with new data structure; display usernames

### PR DESCRIPTION
**Pull request by arringtonh**

### CODE REVIEW

File: proj/src/main/webapp/script.js [hunk 1]
```diff
@@ -362,39 +362,44 @@ function clickWithEnterKey() {
...
-function createMessageWithTemplate(messageObj) {
+function createMessageWithTemplate(key, messageObj) {
     const messageTemplate = document.getElementById('message-temp');
-    const message = messageTemplate.content.cloneNode(true);
+    const docFrag = messageTemplate.content.cloneNode(true);
+    const message = docFrag.querySelector('.message')
 
-    const msgHeader = message.querySelector('.message-header');
-    msgHeader.querySelector('#username').innerText = messageObj.senderDisplay;
+    console.log(messageObj);
```
Comment (showermat) [inline proj/src/main/webapp/script.js]: Does this need to stay?

File: proj/src/main/webapp/script.js [hunk 2]
```diff
@@ -406,18 +411,27 @@ function makeChatPreview(name, messageObj, tag, chatId) {
...
 
     return preview;
 }
 
+// i can't access two paths at the same time so i need two separate functions :/
+function addUsernameToMessage(uid, elementId) {
+    const userRef = firebase.database().ref('/users/'+uid);
+    userRef.once('value', snap => {
+        const element = document.getElementById(elementId);
+        console.log(snap.key)
```
Comment (showermat) [inline proj/src/main/webapp/script.js]: Semicolon for consistency?  Actually, does this line (and the next) need to make it to master at all?

**Review by showermat — APPROVED** [review]
Approved; just left a couple cleanup-related comments.

File: proj/src/main/webapp/script.js (hunk 2 above)
Comment (arringtonh) [reply proj/src/main/webapp/script.js]: fixed!

**Review by nlumiere — APPROVED** [review]
lgtm

**Pull request merged**